src/Albany_Application.cpp

```diff
@@ -735,6 +735,71 @@ Application::finalSetUp(
   for (int i = 0; i < responses.size(); ++i) { responses[i]->postRegSetup(); }
 }
 
+template<typename Traits>
+void
+Application::setSideSetExtents(Teuchos::RCP<PHX::FieldManager<PHAL::AlbanyTraits>>& in_fm) const
+{
+  // get number of worksets
+  const auto& wsElNodeEqID = disc->getWsElNodeEqID();
+  unsigned int const numWorksets = wsElNodeEqID.size();
+
+  // compute largest workset size over all worksets for each sideset name
+  std::map<std::string, unsigned int> maxSideSetSizes;
+  for (unsigned int i = 0; i < numWorksets; ++i) { 
+    const LocalSideSetInfoList& sideSetView = disc->getSideSetViews(i);
+    for (auto it = sideSetView.begin(); it != sideSetView.end(); ++it) {
+      if (maxSideSetSizes.find(it->first) == maxSideSetSizes.end()) {
+        maxSideSetSizes[it->first] = 0;
+      }
+      unsigned int sideSetSize = it->second.size;
```

Comment:
Is it possible for this to be uninitialized? I worry this might be zero and a side field could be set with a zero size. Can we safely add an error here for that case?